client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -93,7 +95,27 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		}
 	};
 
+	receiveMessage = ( event ) => {
+		if ( event.origin && event.origin !== location.origin ) {
+			return;
+		}
+
+		const data = 'string' === typeof event.data
+			? JSON.parse( event.data )
+			: event.data;
```

Comment:
Actually, the response is already JSON. I have no idea why I was trying to parse it. 🤦‍♀️